### DISCUSSION TITLE
Integrazione di Home Assistant con Google Assistant

### DISCUSSION THREAD

**Discussion by basic64v2** [indomus/forum#3327] [HUB: Home Assistant]
Ciao a tutti,
Sono nuovo del forum. Ho installato da poco un nuovo mini-server casalingo e ho voluto provare Home Assistant.
Mi si è aperto un mondo di possibilità. Una di quelle che mi sembrano più interessanti è l'integrazione con Google Assistant, visto che possiedo un Nest Hub.
Ho seguito le istruzioni contenute nella guida ufficiale [https://www.home-assistant.io/integrations/google_assistant/](https://www.home-assistant.io/integrations/google_assistant/)
ma a quanto pare Google ha stoppato il servizio e non è più possibile creare nuove Actions. Eppure su Internet non ho trovato discussioni relative a questo problema e la guida è ancora lì.
Qualcuno è riuscito (in tempi recenti) ad attivare l'integrazione?

**Comment by snakuzzo** [comment]
Va rivista la guida, guarda nei commenti sul sito...ci sono le indicazioni prese da github

**Comment by basic64v2** [comment]
Grazie della dritta. Il messaggio di Google riportato nei commenti parla di migrazione alla developer console. Questo fa sperare che le stesse funzionalità saranno replicabili lì.
Attendo che venga fuori qualche guida o che venga aggiornata quella ufficiale.
